drivers/Makefile.dep

```diff
@@ -0,0 +1,131 @@
+# driver dependencies (in alphabetical order)
+
+ifneq (,$(filter adt7310,$(USEMODULE)))
+  USEMODULE += periph_gpio
+  USEMODULE += periph_spi
+endif
+
+
+ifneq (,$(filter at86rf2%,$(USEMODULE)))
+  USEMODULE += at86rf2xx
+  USEMODULE += ieee802154
+  USEMODULE += periph_gpio
+  USEMODULE += periph_spi
+  ifneq (,$(filter periph_cpuid,$(FEATURES_PROVIDED)))
+    USEMODULE += periph_cpuid
+  endif
+endif
+
```

Comment:
... `undefined reference to `cpuid_get'`. Don't you need to add an entry for the nrfmin driver here or somewhere?